packages/table/src/common/_loading.scss

```diff
@@ -32,7 +32,7 @@
   .pt-skeleton {
     opacity: 0;
     height: $pt-grid-size / 2;
-    animation: skeleton-animation($cell-transition-duration * 2),
-      $cell-transition-duration linear forwards skeleton-fade-in;
+    animation: $skeleton-animation, $cell-transition-duration linear forwards skeleton-fade-in;
+    animation-delay: $cell-transition-duration, 0;
```

Comment:
i would flip the order so the 0-delay transition is first. then it reads in the order it renders.